sormas-api/src/main/java/de/symeda/sormas/api/activityascase/ActivityAsCaseDto.java

```diff
@@ -0,0 +1,271 @@
+ * * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * * GNU General Public License for more details.
+ * *
+ * * You should have received a copy of the GNU General Public License
+ * * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * ******************************************************************************
+ */
+
+package de.symeda.sormas.api.activityascase;
+
+import java.util.Date;
+
+import de.symeda.sormas.api.event.MeansOfTransport;
+import de.symeda.sormas.api.event.TypeOfPlace;
+import de.symeda.sormas.api.exposure.ExposureRole;
+import de.symeda.sormas.api.exposure.GatheringType;
+import de.symeda.sormas.api.exposure.HabitationType;
+import de.symeda.sormas.api.exposure.WorkEnvironment;
+import de.symeda.sormas.api.location.LocationDto;
+import de.symeda.sormas.api.user.UserReferenceDto;
+import de.symeda.sormas.api.utils.DataHelper;
+import de.symeda.sormas.api.utils.HideForCountriesExcept;
+import de.symeda.sormas.api.utils.Required;
+import de.symeda.sormas.api.utils.SensitiveData;
+import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableDto;
+
+public class ActivityAsCaseDto extends PseudonymizableDto {
+
+	private static final long serialVersionUID = 6551672739041643942L;
+
+	public static final String I18N_PREFIX = "ActivityAsCase";
+
+	public static final String REPORTING_USER = "reportingUser";
+	public static final String START_DATE = "startDate";
+	public static final String END_DATE = "endDate";
+	public static final String DESCRIPTION = "description";
+	public static final String ACTIVITY_AS_CASE_TYPE = "activityAsCaseType";
+	public static final String ACTIVITY_AS_CASE_TYPE_DETAILS = "activityAsCaseTypeDetails";
+	public static final String LOCATION = "location";
+	public static final String TYPE_OF_PLACE = "typeOfPlace";
+	public static final String TYPE_OF_PLACE_DETAILS = "typeOfPlaceDetails";
+	public static final String MEANS_OF_TRANSPORT = "meansOfTransport";
+	public static final String MEANS_OF_TRANSPORT_DETAILS = "meansOfTransportDetails";
+	public static final String CONNECTION_NUMBER = "connectionNumber";
+	public static final String SEAT_NUMBER = "seatNumber";
+	public static final String WORK_ENVIRONMENT = "workEnvironment";
+
+	public static final String GATHERING_TYPE = "gatheringType";
+	public static final String GATHERING_DETAILS = "gatheringDetails";
+	public static final String HABITATION_TYPE = "habitationType";
+	public static final String HABITATION_DETAILS = "habitationDetails";
+
+	public static final String EXPOSURE_ROLE = "exposureRole";
+
+	@SensitiveData
+	private UserReferenceDto reportingUser;
+	private Date startDate;
+	private Date endDate;
+	@SensitiveData
+	private String description;
+	@Required
+	private ActivityAsCaseType activityAsCaseType;
+	@SensitiveData
+	private String activityAsCaseTypeDetails;
+	private LocationDto location;
+	@HideForCountriesExcept
+	private ExposureRole exposureRole;
```

Comment:
=> role (because "exposureRole" isn't really applicable anymore, but we definitely don't need a new enum for this)

Reply:
changed